### PR TITLE
monitor_dbus_signals: add messages to interface events

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #89

### CODE REVIEW

File: scripts/monitor_dbus_signals.py [hunk 1]
```diff
@@ -151,6 +155,8 @@ def _properties_changed(*props_changed, object_path=None):
                 for prop in properties_invalidated:
                     data[interface_name][prop] = INVALIDATED
 
+        print("Properties changed.")
```
Comment (mulkieran) [inline scripts/monitor_dbus_signals.py:158]: Indent four spaces...needs to be emitted when ```_properties_changed()``` is invoked.

**Review by mulkieran — CHANGES_REQUESTED** [review]
What we want to do is to log what specific properties changed. This information comes in in the method parameters, and should be able to be printed out relatively easily.

Also, I think the log message code should be moved to the top of each method rather than at the bottom. I'ld rather be logging that the signal was received (at the top) than that processing the signal was completed (at the bottom).

File: scripts/monitor_dbus_signals.py (hunk 1 above)
Comment (bgurney-rh) [reply scripts/monitor_dbus_signals.py]: OK; after that change, I now see this:
```
$ sudo stratis pool create spool1 /dev/vdb1

Interfaces added.
Interfaces added.


$ sudo stratis fs create spool1 sfs1

Interfaces added.
Properties changed.



$ sudo stratis fs destroy spool1 sfs1
Interfaces removed.
Properties changed.


$ sudo stratis pool destroy spool1

Interfaces removed.
Interfaces removed.
```

**Comment by bgurney-rh** [comment]
Rebased.

**Pull request closed without merging**